pkg/util/export/type.go

```diff
@@ -34,6 +34,15 @@ func init() {
 	SetDefaultContextFunc(func() context.Context { return context.Background() })
 }
 
+const (
```

Comment:
I have seen this in many places.   While it is fine, maybe we should use the one in Common.

Reply:
got it.
I will `pkg/common/constant/byte.go` handle const value `Byte, KB, MB, GB, ...`.
and try to handle other pkgs in new pr.

Reply:
i find `KB, MB, GB, TB, ...` already defined in `pkg/common/mpool/mpool.go`
should i just use it, or mv it to `pkg/common/constant/byte.go` ?

Reply:
You can move it, your call. :)